sdk/Sdk.Generators/Sdk.Generators.csproj

```diff
@@ -10,6 +10,7 @@
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <VersionSuffix>-preview2</VersionSuffix>
+    <IsRoslynComponent>true</IsRoslynComponent>
```

Comment:
Trying to get #1191 merged before this and will rebase after - leaving these changes checked in for now in case anyone wants to try this out locally.